tests/integration/__init__.py

```diff
@@ -1,5 +1,22 @@
+from requre.helpers.requests_response import RequestResponseHandling
 from requre.import_system import upgrade_import_system
-from tests.replacements import MODULE_LIST
 
-
-upgrade_import_system(MODULE_LIST, debug_file="modules.out")
+ogr_import_system = upgrade_import_system(
+    filters=[
+        ("^requests$", {"who_name": "ogr"}),
+        ("^requests$", {"who_name": "gitlab"}),
+        ("^requests$", {"who_name": "github"}),
```

Comment:
This can be improved after merging this: https://github.com/packit-service/requre/pull/36